client/my-sites/promote-post-i2/components/campaign-item-details/index.tsx

```diff
@@ -875,7 +879,7 @@ export default function CampaignItemDetails( props: Props ) {
 														label={ chartSource }
 													/>
 													{ getCampaignStatsChart(
-														campaignStats?.series[ chartSource ] ?? [],
+														campaignStats?.series[ chartSource ] ?? null,
```

Comment:
empty array can be a valid response for no data, so I intend to handle these cases differently